zktraffic/fle/message.py

```diff
@@ -0,0 +1,133 @@
+# ==================================================================================================
+# Copyright 2014 Twitter, Inc.
+# --------------------------------------------------------------------------------------------------
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this work except in compliance with the License.
+# You may obtain a copy of the License in the LICENSE file, or at:
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==================================================================================================
+
+from datetime import datetime
+
+from zktraffic.base.network import BadPacket
+from zktraffic.base.util import read_long, read_number, read_string
```

Comment:
I prefer doing multiple imports this way:

from foo.bar import {
  baz,
  biz
}

At least I think it works that way. Makes diffs really easy to read.